src/AppInstallerSharedLib/Errors.cpp

```diff
@@ -222,6 +222,8 @@ namespace AppInstaller
                 return "One or more queries did not return exactly one match";
             case APPINSTALLER_CLI_ERROR_PACKAGE_IS_PINNED:
                 return "The package has a pin that prevents upgrade.";
+            case APPINSTALLER_CLI_ERROR_PACKAGE_IS_STUB:
+                return "The package currently installed is the stub package";

```

Comment:
This should also be added to `doc/windows/package-manager/winget/returnCodes.md`